umpleonline/scripts/umpleCollab.js

```diff
@@ -22,8 +22,8 @@ let inactivitywarningTime = 540000; // 9 minutes
 let diff = (inactivityTime - inactivitywarningTime)/1000; // 60 seconds
 let inactivityTimer; // Variable to store the timer
 let inactivitywarningTimer;
-let debugFlag = false; 
-let achknowledgedTimer = 0; 
+let collabDebugFlag = false; 
```

Comment:
We can name this variable **collabClientDebugFlag** - just to signify it is part of client side